pages/about.html

```diff
@@ -0,0 +1,139 @@
+                              <a class="nav-link" href="#">Join</a>
+                          </li>
+                          <li class="nav-item">
+                              <a class="nav-link" href="#">Sponsor</a>
+                          </li>
+                          <li class="nav-item">
+                              <a class="nav-link" href="#">News</a>
+                          </li>
+                          <li class="nav-item">
+                              <a class="nav-link" aria-current="page" href="../index.html">Home</a>
+                          </li>
+                      </ul>
+                  </div>
+              </nav>
+          </div>
+      </header>
+        <section class="hello_section flex-column">
+          <div class="welcome_heading container text-center">
+            <h1>Your delicious snacks</h1>
+            <hr class="line mb-5">
+        </div>
+        <div class="bg-light container">
+          <p class="m-5 text">
+            The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network 
+            in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals who want 
+            to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge. So if you're active and engaged 
+            in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, 
+            governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons 
+            together
+          </p>
+        </div>
+        <div class="email">
+          <p>
+            Please contact us per Email for nay further questions about CC Global Summit 2015!
+          </p>
+          <span>ccsummit2015@cckorea.org</span>
+        </div>
+        </section>
+        <section class="summit">
+          <div class="container text-center">
+            <h3>Creative Commons Global Summit 2015 Logo</h3>
+            <hr class="line mb-5">
+            <p>
+              The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+              After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+              After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+            </p>
+            <img src="../assets/images/about_logo1.png" alt="About logo">
+          </div>
+          <div class="container my-5 text-center">
+            <h3>See the past CC Global Summits</h3>
+            <hr class="line mb-5">
+            <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
```

Comment:
- [ ] the footer of the about page has a dark background.
- [ ] please the desktop version partners section is not an image but logos of the different partners. Kindly fix that